CHANGELOG.md

```diff
@@ -1,71 +1,121 @@
+ [#597](https://github.com/mattermost/desktop/pull/597)
+
 ### Bug Fixes
 
 #### All Platforms
- - Fixed mis-aligned `+` button of tab bar.
- [#541](https://github.com/mattermost/desktop/issues/541)
  - Fixed the close button of the Settings page not working on first installation.
  [#552](https://github.com/mattermost/desktop/issues/552)
- - Fixed the app publisher was not changed to Mattermost, Inc.
+ - Fixed the app publisher referring to Yuya Ochiai instead of Mattermost, Inc.
  [#542](https://github.com/mattermost/desktop/issues/542)
- - Fixed an extra reloading when an error page is manually reloaded.
+ - Fixed font size not always persisting across app restarts.
+ [#564](https://github.com/mattermost/desktop/issues/564)
+ - Fixed an automatic reloading of the app when a DNS or network error page is manually reloaded with CTRL/CMD+R.
  [#573](https://github.com/mattermost/desktop/issues/573)
- - Fixed file upload dialogs did not allow any file to be selected.
- [#497](https://github.com/mattermost/desktop/issues/497)
- - Fixed an issue where unnecessary focus remains after closing dialogs on the settings page.
- [#446](https://github.com/mattermost/desktop/issues/446)
- - Fixed an issue where there is a case of that modified font size causes wrong rendering.
+ - Fixed an issue where changing font size caused rendering issues on next restart.
  [#334](https://github.com/mattermost/desktop/issues/334)
+ - Fixed an issue where after adding a server on the Settings page, focus remained on the "Add new server" link.
+ [#446](https://github.com/mattermost/desktop/issues/446)
+ - Fixed an issue where SAML certificate file couldn't be uploaded from the file upload dialog.
+ [#497](https://github.com/mattermost/desktop/issues/497)
 
 #### Windows
- - Fixed desktop notifications not working when the window has been minimized from inactive state.
+ - Fixed desktop notifications not working when the window was minimized from an inactive state.
  [#522](https://github.com/mattermost/desktop/issues/522)
- - Fixed the uninstaller not removing files correctly.
+ - Fixed the uninstaller not removing all files correctly.
  [#551](https://github.com/mattermost/desktop/issues/551)
 
 #### Mac
- - Fixed an issue where the text box didn't keep focus after uploading a file.
+ - Fixed an issue where after uploading a file, focus wasn't put back to the text box.
  [#341](https://github.com/mattermost/desktop/issues/341)
+ - Fixed a mis-aligned `+` button in the server tab bar.
+ [#541](https://github.com/mattermost/desktop/issues/541)
 
 #### Linux
- - Fixed the main window not minimized when the app is launched via "Start app on Login" option.
+ - Fixed the main window not being minimized when the app is launched via "Start app on Login" option.
  [#570](https://github.com/mattermost/desktop/issues/570)
 
+### Known Issues
+
```

Comment:
Should we put "None" or "N/A" under this heading just so it isn't floating around in space?

Reply:
There are known issues for All Platforms, and Windows/Mac/Linux separately. So I think it's appropriate to have this heading